Tasks/Gradle/CodeAnalysis/SonarQube/server.ts

```diff
@@ -26,24 +26,29 @@ export class SonarQubeServer implements ISonarQubeServer {
     }
 
     public invokeApiCall(path:string):Q.Promise<Object> {
-        var defer = Q.defer<Object>();
 
+        tl.debug(`[SQ] Calling a REST API: ${path}`);
+
+        var defer = Q.defer<Object>();
         var options:any = this.createSonarQubeHttpRequestOptions(path);
 
         // Dynamic switching - we cannot use https.request() for http:// calls or vice versa
         var protocolToUse;
         switch (options.protocol) {
-            case 'http':
+            case 'http:':
```

Comment:
The original reason we didn't use the request package was because of testability issues. I don't think that problem is still extant.